model/event/event_test.go

```diff
@@ -280,6 +283,33 @@ func (s *eventSuite) TestFindResourceTypeIn() {
 	found, data = findResourceTypeIn(nil)
 	s.False(found)
 	s.Empty(data)
+
+	s.NotPanics(func() {
```

Comment:
Question: does this line suppress all panics? Shouldn't it be defined for some specific cases (not all)?

Reply:
It runs the provided function, and if any panic occurred, the test fails. On the flip side, you can also use PanicsWithValue to test for specific panics.

Reply:
@unrealsolver https://github.com/stretchr/testify/blob/master/assert/assertions.go#L903-L916